### DISCUSSION TITLE
"unhardcode" Round Announcement

### DISCUSSION THREAD

**Discussion by Amara-MFG** [ikemen-engine/Ikemen-GO#1995] [Feature Request]
### Is your feature request related to a problem? Please describe.

I'm planning on making a screenpack for IKEMEN and I've been pitching some ideas. Unfortunately, I can't have the current round be announced BEFORE the character intros (like in KOF98 for example) without any workarounds with explods on characters themselves because this is hardcoded in the engine apparently.

It would be nice if we could customize it how we want so it isn't dependant on coding the characters or having to go through hassles on workarounds.

### Describe the solution you'd like

Have it so you can adjust the timer when the round gets announced (e.g.: before or after character intros - say 0 for before and 1 for after).

### Additional context

https://github.com/user-attachments/assets/811e0f60-f04a-49a4-9c38-44586d454a36

This sums it up.

**Comment by potsmugen** [comment]
As mentioned on Discord it's worth keeping in mind that such a feature would not be compatible with many (most?) existing characters. But it sounds like something we should implement at some point, yeah.

**Reply by Amara-MFG** [reply]
What if we hardcode it so they‘re frozen on the frame 1 of their intro until the round is announced? Would that work?

**Reply by potsmugen** [reply]
That sounds very ironic, heh.

**Reply by Amara-MFG** [reply]
True HAHAHAHA
